# This message is used to send a command with local leg positions to
# the inverseKinematicsAdapter.

geometry_msgs/Point[4] legPosition
